java/mykola-melnyk/src/com/zoolatech/lecture5/tasks/_3/CaseAgnosticComparator.java

```diff
@@ -0,0 +1,14 @@
+package com.zoolatech.lecture5.tasks._3;
+
+import java.util.Comparator;
+
+public class CaseAgnosticComparator implements Comparator<String> {
```

Comment:
Nice idea. Just a note that such comparator is already provided by class String: String.CASE_INSENSITIVE_ORDER

Reply:
Thank you, will know that :)